packages/pcre.rb

```diff
@@ -8,6 +8,8 @@ class Pcre < Package
   source_url 'https://ftp.pcre.org/pub/pcre/pcre-8.44.tar.bz2'
   source_sha256 '19108658b23b3ec5058edc9f66ac545ea19f9537234be1ec62b714c84399366d'
 
+  depends_on 'readline'
```

Comment:
No need to include core package.

Reply:
Does it hurt to do so? I feel like it's better to be explicit about which packages depend on each-other, core package or no. Certainly right now we can assume all core packages are present on the system, but what about tomorrow or next year?

Reply:
Removed it as requested.